### PR TITLE
Add adopters list

### PR DESCRIPTION
**Pull request by yurishkuro**
rephrased version of #332

### CODE REVIEW

**Comment by caniszczyk** [comment]
LGTM now, the language is better now, thanks

File: README.md [hunk 1]
```diff
@@ -53,6 +53,16 @@ Reach project contributors via these channels:
  * [Gitter chat room](https://gitter.im/jaegertracing/Lobby)
  * [Github issues](https://github.com/uber/jaeger/issues)
 
+## Adopters
+
+Jaeger as a product consists of multiple components. We want to support different types of users,
```
Comment (vprithvi) [inline README.md]: I'd rephrase this.

Something like

Jaeger consists of many components, and we count users of at least one of these as adopters. 
Users may lie on a spectrum ranging from only using Jaeger's instrumentation libraries to running Jaeger end-to-end. They may be running it on a Internet facing production environment, or a testing or local development environment to troubleshoot issues.

**Comment by adriancole** [comment]
nice job!
